source/elements/oneDAL/source/algorithms/modes.rst

```diff
@@ -0,0 +1,17 @@
+Computation modes
+=========================
+
+The library algorithms support the following computation modes:
+
+    Batch processing
+    Online processing
+    Distributed processing
+
+
+
+.. toctree::
+   :maxdepth: 3
+
+   modes/batch.rst
+   modes/online.rst
+   modes/distributed.rst
```

Comment:
I'd suggest you only leave the toctree here as it creates a list of computation modes.
![image](https://user-images.githubusercontent.com/17865025/76161087-043b8700-6141-11ea-8475-8f90260ae4f2.png)


If you want to have a list from lines 6-8, you need to rewrite it like this:

```
- Batch processing
- Online processing
- Distributed processing
```

Reply:
Fixed, thanks.